UI/GovernmentWnd.cpp

```diff
@@ -416,14 +412,21 @@ const std::set<std::string>& PoliciesListBox::GetCategoriesShown() const
 void PoliciesListBox::SizeMove(const GG::Pt& ul, const GG::Pt& lr) {
     GG::Pt old_size = GG::Wnd::Size();
 
+    double zoom_factor = 1;
```

Comment:
could probably make a separate function for getting the policy width and height as a pair, rather than repeating this and similar code in a few places

Reply:
I did this, it doesn't save much space but looks cleaner.